oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/BearerTokenAuthenticationFilter.java

```diff
@@ -152,4 +156,13 @@ public final void setAuthenticationEntryPoint(final AuthenticationEntryPoint aut
 		Assert.notNull(authenticationEntryPoint, "authenticationEntryPoint cannot be null");
 		this.authenticationEntryPoint = authenticationEntryPoint;
 	}
+
+	/**
+	 * Set the {@link AuthenticationFailureHandler} to use. Default implementation invokes {@link AuthenticationEntryPoint}.
+	 * @param authenticationFailureHandler the {@code AuthenticationFailureHandler} to use
```

Comment:
@ThomasVitale will you please add a `@since 5.2` marker here?

Reply:
@jzheaux I've added it